spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/GlobalAveragePooling1D.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.nn.keras
+
+import com.intel.analytics.bigdl.nn.abstractnn._
+import com.intel.analytics.bigdl.nn._
+import com.intel.analytics.bigdl.nn.SpatialAveragePooling
+import com.intel.analytics.bigdl.nn.{Sequential => TSequential}
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+import com.intel.analytics.bigdl.utils.Shape
+
+import scala.reflect.ClassTag
+
+class GlobalAveragePooling1D[T: ClassTag](inputShape: Shape = null
+                                         )(implicit ev: TensorNumeric[T])
+  extends GlobalPooling1D[T](inputShape) {
+
+  override def doBuild(inputShape: Shape): AbstractModule[Tensor[T], Tensor[T], T] = {
+    val input = inputShape.toSingle().toArray
+
+    val model = TSequential[T]()
+    model.add(Reshape(Array(input(1), 1, input(2)), Some(true)))
+
+    val layer = SpatialAveragePooling(
+      kW = 1,
+      kH = input(1),
+      dW = 1,
+      dH = 1,
+      padW = 0,
+      padH = 0,
+      globalPooling = false,
+      ceilMode = false,
+      countIncludePad = false,
+      divide = true,
+      format = DataFormat.NHWC
+    )
+
```

Comment:
Remove unnecessary blank lines. Also do this in other class files if there are any.